imports/ui/Login.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import AccountsUIWrapper from './AccountsUIWrapper.jsx';
+
+export default class Login extends Component {
+  render() {
+    return (
+
+        <header className="login">
+        <h1>LOGIN</h1>
```

Comment:
Uwaga! To jest testowy komentarz! Np. There's a trailing whitespace here, please remove it. (Dodana pusta spacja, proszę usuń ;) )